db/migrate/20150714132601_remove_is_param.rb

```diff
@@ -0,0 +1,9 @@
+class RemoveIsParam < ActiveRecord::Migration
+  def up
+    remove_column :lookup_keys, :is_param
+  end
+
+  def down
+    add_column :lookup_keys, :is_param, :boolean
+  end
+end
```

Comment:
yes but not here it should be in: https://github.com/theforeman/foreman/pull/2466/files#diff-7602204a9c75f255b4f52a44636017a8R1